Sources/BranchSDK/Branch.m

```diff
@@ -2107,6 +2107,8 @@ - (void)handleInitSuccessAndCallCallback:(BOOL)callCallback sceneIdentifier:(NSS
         id<NSObject> sharedApplication = [applicationClass performSelector:@selector(sharedApplication)];
         if ([sharedApplication respondsToSelector:@selector(openURL:)])
             [sharedApplication performSelector:@selector(openURL:) withObject:comp.URL];
+    } else if ([latestReferringParams[@"validate_integration"] isEqualToString:@"true"]) {
+        [self validateSDKIntegration];
```

Comment:
If deep linking fails completely, this will never trigger which is an outcome we are fine with. But there are multiple entry points to this function - let's say universal link launch fails, it will launch via URI scheme. If both of these fail, developer can try with the alternate link domain. The idea is that if this fails to load completely, we still have some understanding that the core issue is with the universal links setup itself. 

As of today, debugging an issue is a blind spot for CX teams because the error could be anywhere. This function helps in breaking down the issue into (a). either the issue is fundamentally with the links setup (if the validator doesn't launch at all) or (b). the issue is one of the issues highlighted by the validator.